apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/AddBlock/NewTextResponseButton/NewTextResponseButton.tsx

```diff
@@ -32,12 +42,20 @@ export const TEXT_RESPONSE_BLOCK_CREATE = gql`
 
 export function NewTextResponseButton(): ReactElement {
   const { t } = useTranslation('apps-journeys-admin')
-  const [textResponseBlockCreate, { loading }] =
+  const [textResponseBlockCreate, { loading: textResponseLoading }] =
     useMutation<TextResponseBlockCreate>(TEXT_RESPONSE_BLOCK_CREATE)
+
+  const [create, { loading: textResponseWithButtonLoading }] =
```

Comment:
```suggestion
  const [textReponseWithButtonCreate, { loading: textResponseWithButtonLoading }] =
```

thoughts on this instead? makes it easier to follow further down this file. more descriptive than create